.travis.yml

```diff
@@ -0,0 +1,25 @@
+env:
+  - NODE_VERSION=0.10
+  - NODE_VERSION=0.12
+
+os:
+  - linux
+  - osx
+
+install:
+  - git clone https://github.com/creationix/nvm.git /tmp/.nvm
+  - source /tmp/.nvm/nvm.sh
+  - nvm install $NODE_VERSION
+  - nvm use $NODE_VERSION
```

Comment:
Just curious; why did you choose to install node.js like this, rather than w/ travis's node support?

```yaml
node_js:
  - "0.12"
  - "0.10"
```

Does builtin node support not work w/ multi-os builds?